test/cpp/interop/client_helper.cc

```diff
@@ -84,13 +84,11 @@ std::shared_ptr<Channel> CreateChannelForTestCase(
     std::vector<
         std::unique_ptr<experimental::ClientInterceptorFactoryInterface>>
         interceptor_creators) {
-  GPR_ASSERT(absl::GetFlag(FLAGS_server_port));
-  const int host_port_buf_size = 1024;
-  char host_port[host_port_buf_size];
-  snprintf(host_port, host_port_buf_size, "%s:%d",
-           absl::GetFlag(FLAGS_server_host).c_str(),
-           absl::GetFlag(FLAGS_server_port));
-
+  std::string host_port = absl::GetFlag(FLAGS_server_host);
```

Comment:
why not just call this host instead of host_port?

Reply:
What do you think about server_uri? (changed to that)

Preferred that over "host" since host may sound like it includes only the hostname component of the uri